src/Map/doc/index.rst

```diff
@@ -136,10 +136,37 @@ You can add markers to a map using the ``addMarker()`` method::
         ))
     ;
 
+Remove elements from Map
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+It is possible to remove elements like ``Marker``, ``Polygon`` and ``Polyline`` instances by using methods ``Map::remove*()``:
+    // Add elements
+    $map->addMarker($marker = new Marker(/* ... */));
+    $map->addPolygon($polygon = new Polygon(/* ... */));
+    $map->addPolyline($polyline = new Polyline(/* ... */));
+    
+    // And later, remove those elements
+    $map->removeMarker($marker);
+    $map->removePolygon($polygon);
+    $map->removePolyline($polyline);
+    
+If unfortunately you were unable to store an element instance, you can still remove them by passing the identifier string:
+ 
+    $map = new Map(/* ... */);
+    // Add elements
+    $map->addMarker(new Marker(id: 'my-marker', /* ... */));
+    $map->addPolygon(new Polygon(id: 'my-polygon', /* ... */));
+    $map->addPolyline(new Polyline(id: 'my-marker', /* ... */));
+    
+    // And later, remove those elements
+    $map->removeMarker('my-marker');
+    $map->removePolygon('my-polygon');
+    $map->removePolyline('my-marker');
+
 Add Polygons
 ~~~~~~~~~~~~
 
-You can also add Polygons, which represents an area enclosed by a series of ``Point`` instances::
+You can also add Polygons, which represents an area enclosed by a series of ``Point`` instances:
```

Comment:
To revert.

Please keep `::`, it is needed to correctly [highlight PHP code blocks in Symfony documentation](https://symfony.com/doc/current/contributing/documentation/standards.html#sphinx)